framework/source/class/qx/test/bom/Attribute.js

```diff
@@ -20,7 +20,7 @@
 ************************************************************************ */
 /**
  *
- * @asset(qx/icon/${qx.icontheme}/48/places/folder.png)
+ * @asset(qx/icon/Tango/48/places/folder.png)
```

Comment:
This looks like a reverse diff. The stuff being added is Tango, with qx.icontheme being removed.

Reply:
This is correct. I reverted the asserts for the unit tests.
Later in the unit test code  qx/icon/Tango/48/places/folder.png is used for testing.